paasta_tools/kubernetes/application/controller_wrappers.py

```diff
@@ -411,14 +413,63 @@ def update(self, kube_client: KubeClient):
         )
 
 
+class JobWrapper(Application):
+    def __init__(
+        self,
+        item: V1Job,
+        logging=logging.getLogger(__name__),
+    ):
+        item.spec.replicas = None
```

Comment:
i assume this is here so that we don't get an AttributeError in `sanitize_for_config_hash()`?

might be worth commenting this line so that an enterprising yelper doesn't try to be "helpful" and clean this up in the future :)

Reply:
good point